### DISCUSSION TITLE
Clickhouse

### DISCUSSION THREAD

**Discussion by pistoletov1974** [fluvio-community/fluvio#3984] [Ideas]
Hi! do you plan to add [Clickhouse](https://clickhouse.com/) database support?  This is exelent database for data analytics.

**Comment by drc-infinyon** [comment]
Yes @pistoletov1974 we spoke with ClickHouse folks about this a month back. We have plans to build a ClickHouse connector.

Can you share more about your idea?
Are you looking to write data from Fluvio topics into ClickHouse and you are looking for an outbound sink connector to write to ClickHouse?
Or, do you intend to read from ClickHouse into Fluvio topics?
Or, do you intend to do both?

Let me know.

**Reply by pistoletov1974** [reply]
I looking for write data from fluvio topic into Clickhouse(outbound connector). 
I suggest adding Сlickhouse   support to Fluvio SQL Sink connector

**Comment by prashis** [comment]
Hi, I came across this thread while searching for a way to push data from Fluvio topics to Clickhouse. Is there any solution for this?

**Comment by drc-infinyon** [comment]
We have a couple of live integrations in ingest into click house. It's not on the InfinyOn hub yet. @prashis can you share more about your use case, and how urgent is your requirement.

**Reply by prashis** [reply]
I was looking for something similar as https://github.com/ClickHouse/clickhouse-kafka-connect. Not urgent as of now.

**Reply by drc-infinyon** [reply]
Sounds good @prashis. Let me connect with the team and see what we can do.

Fluvio connectors are built using CDK - https://www.fluvio.io/docs/connectors/developers/overview And published on the Hub.

Created an issue to track this request - https://github.com/infinyon/fluvio/issues/4279

I will work with the team and initialize the repository for a ClickHouse Connector and share.

Community contributions are welcome.

**Reply by prashis** [reply]
Thank you for creating an issue & sharing relevant documentation. I will go through the documentation & if possible I'll contribute.

**Reply by sehz** [reply]
We have customers who created their own click house connector and used SDF and InfinyOnCloud's webhook  to live feed data. We will share example of how to do that.

Overview of SDF: https://www.fluvio.io/sdf/ and Cloud https://www.fluvio.io/docs/cloud/quickstart.
